bundles/org.openhab.binding.mecmeter/src/main/java/org/openhab/binding/mecmeter/internal/api/MecMeterResponse.java

```diff
@@ -0,0 +1,856 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.mecmeter.internal.api;
```

Comment:
SAT warnings about missing `@NonNullByDefault` can be prevented by moving it to another package:

```suggestion
package org.openhab.binding.mecmeter.internal.dto;
```

Reply:
Didn't know, thanks. Moved.